firefox_puppeteer/ui/windows.py

```diff
@@ -99,6 +99,9 @@ def create_window_instance(self, handle, expected_class=None):
 
         if window_type == 'navigator:browser':
             window = BrowserWindow(lambda: self.marionette, handle)
+        elif window_type == 'Browser:page-info':
+            from .pageinfo.window import PageInfoWindow
```

Comment:
This is still the same issue as we talked about on another PR a couple of weeks ago. But I will check it again later.